moveit_ros/planning/planning_scene_monitor/demos/demo_scene.cpp

```diff
@@ -75,23 +84,27 @@ void sendKnife()
   co.primitive_poses[0].position.z = -0.2;
   co.primitive_poses[0].orientation.w = 1.0;
 
-  pub_aco.publish(aco);
+  pub_aco->publish(aco);
   sleep(1);
-  pub_aco.publish(aco);
-  ROS_INFO("Object published.");
-  ros::Duration(1.5).sleep();
+  pub_aco->publish(aco);
+
+  RCLCPP_INFO(node->get_logger(), "Object published.");
+  rclcpp::sleep_for(std::chrono::milliseconds(1500));
 }
 
 int main(int argc, char** argv)
 {
-  ros::init(argc, argv, "demo", ros::init_options::AnonymousName);
+  signal(SIGINT, signalHandler);
 
-  ros::AsyncSpinner spinner(1);
-  spinner.start();
+  rclcpp::init(argc, argv);
+  rclcpp::executors::SingleThreadedExecutor executor;
 
-  sendKnife();
+  node = rclcpp::Node::make_shared("demo");
 
-  ros::waitForShutdown();
+  while (!shutdown_req){
+  sendKnife();
```

Comment:
Why's this not tabbed appropriately @anasarrak? Please have a look at https://moveit.ros.org/documentation/contributing/code/. Particularly, `clang-format Auto Code Formatting`. While I'm not sure what's their policy for accepting changes here (they complained a few times when I runned it as they requested), I think it's a good practice we should also apply.

Reply:
Fine, I used the following command to format just a single file
```
find . -name moveit_ros/planning/planning_scene_monitor/demos/demo_scene.cpp | xargs clang-format-3.9 -i -style=file $1
```
The result: https://github.com/AcutronicRobotics/moveit2/pull/39/commits/6eb5254eaed0b35328dd982907afe2825cf777e5